@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -104,6 +104,10 @@ export default {
       type: String,
       default: 'col-12 col-md-6'
     },
+    showPromotionalTimer: {
+      type: Boolean,
+      default: true
+    },
```

Comment:
```suggestion
    hasPromotionTimer: Boolean,
```

Em APIs de componentes naming things é importante, booleano sempre com `has`, `can` ou `is`.
O padrão dessa prop deve ser false (pelo menos por enquanto).

Reply:
Resolvido